packages/aws-cdk/lib/commands/flags/interactive-handler.ts

```diff
@@ -0,0 +1,84 @@
+import type { FeatureFlag } from '@aws-cdk/toolkit-lib';
+// @ts-ignore
+import { Select } from 'enquirer';
+import type { FlagOperations } from './operations';
+import { FlagsMenuOptions, type FlagOperationsParams } from './types';
+
+export class InteractiveHandler {
+  constructor(
+    private readonly flags: FeatureFlag[],
+    private readonly flagOperations: FlagOperations,
+  ) {
+  }
+
+  /** Displays flags that have differences between user and recommended values */
+  private async displayFlagsWithDifferences(): Promise<void> {
+    const flagsWithDifferences = this.flags.filter(flag =>
+      flag.userValue === undefined || !this.isUserValueEqualToRecommended(flag));
+
+    if (flagsWithDifferences.length > 0) {
+      await this.flagOperations.displayFlagTable(flagsWithDifferences);
+    }
+  }
+
+  /** Checks if user value matches recommended value */
+  private isUserValueEqualToRecommended(flag: FeatureFlag): boolean {
+    return String(flag.userValue) === String(flag.recommendedValue);
+  }
+
+  /** Main interactive mode handler that shows menu and processes user selection */
+  async handleInteractiveMode(): Promise<FlagOperationsParams | null> {
```

Comment:
"Handler" and "handle" ?

I think some more specific terminology we could use is "prompt user for command".